docs/scos/dev/glue-api-guides/202204.0/glue-backend-api/authentication-and-authorization.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Authentication and authorization
+description: Information about Glue API authentication and authorization.
+last_updated: Oct 23, 2022
+template: glue-api-storefront-guide-template
```

Comment:
[Informational] We have both `storefront` and `backend` API in one document, which is located in `backend-api` section. This should be refactored in the following PR's. CC @gechetspr

Reply:
Yes, we discussed it with @gechetspr